docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -0,0 +1,142 @@
+
+In this guide, for ease of explanation, we deploy Auditor in a different cluster on the same subscription. However, it is highly recommended to deploy it in another administrative domain in production.
+
+### Recommendations
+
+* You should deploy Ledger and Auditor in separate administrative domains.
+
+### Steps
+
+* Follow step 1 to step 3 in the [Deploy Scalar DL on AWS](./ManualDeploymentGuideScalarDLOnAWS.md) guide to create an Scalar DL Auditor environment.
+
+## Step 2. Peer the Virtual Networks
+
+This section shows how to peer the VPC for Scalar DL deployment.
+
+In this guide, Ledger and Auditor are deployed on the private subnet of the different VPCs, 
+so you need to add peering for internal communication between the Auditor, Ledger and Client SDK(application).  
+
+### Requirements
+
+* You must create a peering between the VPC created for Ledger and Auditor services.
+* You must create a peering between the VPC created for Ledger and Client SDK (application).
+* You must create a peering between the VPC created for Auditor and Client SDK (application).
+
+### Steps
+
+* Create the peering on the basis of [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html) with the above requirements.
+* Update your all route tables for a VPC peering connection on the basis of [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html#describe-vpc-peering-connections).
+
+## Step 3. Install Scalar DL Auditor
+
+### Prerequisites
+
+You must install Helm on your bastion to deploy [helm-charts](https://github.com/scalar-labs/helm-charts):
+
+* [Helm](https://helm.sh/docs/intro/install/): Helm command-line tool to manage releases in the AKS cluster, Helm version 3.2.1 or latest is required. In this guide, it is used to deploy Scalar Auditor and Schema loading helm charts to the AKS cluster.
+* You must create a Github Personal Access Token (PAT) on the basis of Github official document with read:packages scope, it is used to access the `scalar-auditor` and `scalardl-schema-loader` container images from GitHub Packages.
+
+### Requirements
+
+* You must have the authority to pull `scalar-auditor` and `scalardl-schema-loader` container images.
+
+### Recommendations
+
+* You should confirm that the replica count of the Auditor and Envoy pods in the `scalardl-audit-custom-values.yaml` file is equal to the number of nodes in the scalardlpool.
+* You should keep an equal number of pods for Envoy, Ledger and Auditor.
+
+### Steps
+
+1. Download the following Scalar DL configuration files from the [scalar-kubernetes](https://github.com/scalar-labs/scalar-kubernetes/tree/audit-guide-aks/conf) repository.
+Note that they are going to be versioned in the future, so you might want to change the branch to use a proper version.
+    * Scalardl-audit-custom-values.yaml
+    * Schema-loading-custom-values.yaml
```

Comment:
```suggestion
    * scalardl-audit-custom-values.yaml
    * schema-loading-custom-values.yaml
```

Reply:
Thank you for your suggestions. 
Document formatted.